packages/victory-tooltip/src/victory-tooltip.tsx

```diff
@@ -159,7 +146,7 @@ export default class VictoryTooltip extends React.Component {
   getPolarOrientation(props, datum) {
```

Comment:
I didn't type most of these helper functions, since doing so mostly led to extraneous null checks or would require creating a new type to represent the evaluated, defaulted props, which didn't appear to be a pattern elsewhere.